src/controllers/controller.js

```diff
@@ -378,6 +379,36 @@ class Controller extends EventEmitter {
 		return addr
 	}
 
+	async getIntersection(latitude, longitude) {
+		if (this.config.geocoding.intersection_users == [""]){
+			return 'No Intersection'
```

Comment:
would need to provide a translation lookup (function could have translator passed in)

Reply:
Was able to fix everything else, not really sure how to get the translation in.